getting-started/README.md

```diff
@@ -1,110 +1,108 @@
 
-- GenAIComps is a collection of microservice components that form a service-based toolkit. This includes a variety of services such as llm (language learning models), embedding, and reranking, among others.
-- While GenAIComps offers a range of microservices, GenAIExamples provides practical, deployable solutions to help users implement these services effectively. Examples include ChatQnA and DocSum, which leverage the microservices for specific applications.
+5. Create or add an ssh key for the instance.
 
-## Visual Guide to Deployment
-To illustrate, here's a simplified visual guide on deploying a ChatQnA GenAIExample, showcasing how you can set up this solution in just a few steps.
+6. Click on `Create virtual server` at the bottom right.
 
-![Getting started with OPEA](assets/getting_started.gif)
+7. Once the instance is running, create and attach a `Floating IP` to the instance. For more information visit [this](https://cloud.ibm.com/docs/vpc?topic=vpc-fip-working&interface=ui) site
 
-## Setup ChatQnA Parameters
-To deploy ChatQnA services, follow these steps:
+8. `ssh` into the instance using the floating IP (`ssh -i <key> ubuntu@<floating-ip>`)
 
+
+
+## Deploy the ChatQnA Solution
+Use the command below to install docker on a clean virtual machine
+```bash
+wget https://raw.githubusercontent.com/opea-project/GenAIExamples/refs/heads/main/ChatQnA/docker_compose/install_docker.sh
+chmod +x install_docker.sh
+./install_docker.sh
 ```
+Configure Docker to run as a non-root user by following these [instructions](https://docs.docker.com/engine/install/linux-postinstall/)
+
+Clone the repo and navigate to ChatQnA
+
+```bash
 git clone https://github.com/opea-project/GenAIExamples.git
 cd GenAIExamples/ChatQnA
 ```
-### Set the required environment variables:
-```
-# Example: host_ip="192.168.1.1"
-export host_ip="External_Public_IP"
-# Example: no_proxy="localhost, 127.0.0.1, 192.168.1.1"
-export no_proxy="Your_No_Proxy"
+Set the required environment variables:
+```bash
+export host_ip="localhost"
 export HUGGINGFACEHUB_API_TOKEN="Your_Huggingface_API_Token"
 ```
-If you are in a proxy environment, also set the proxy-related environment variables:
-```
-export http_proxy="Your_HTTP_Proxy"
-export https_proxy="Your_HTTPs_Proxy"
-```
-
-Set up other specific use-case environment variables by choosing one of these options, according to your hardware:
 
+Set up other specific use-case environment variables:
+```bash
+cd docker_compose/intel/cpu/xeon/
+source set_env.sh
 ```
-# on Xeon
-source ./docker_compose/intel/cpu/xeon/set_env.sh
-# on Gaudi
-source ./docker_compose/intel/hpu/gaudi/set_env.sh
-# on Nvidia GPU
-source ./docker_compose/nvidia/gpu/set_env.sh
+Now we can start the services
+```bash
+docker compose up -d
 ```
+It takes a few minutes for the services to start. Check the logs for the services to ensure that ChatQnA is running.
+For example to check the logs for the `tgi-service`:

```

Comment:
Check if all the services are up and running. 
> docker ps -a
Expect the following container images:

<img width="1320" alt="image" src="https://github.com/user-attachments/assets/67ba9639-5513-4452-9a53-8d427cd09dc2">